commit dd93fc4bfbaceb65ded2aefca35042438c95f4d5
Author: Justus Winter <justus@gnupg.org>
Date:   Fri Jun 3 00:45:11 2016 +0200

    nfs: appease the stricter reference counting mechanism
    
    * nfs/cache.c (netfs_node_norefs): Do not re-acquire a reference to
    np.  This worked previously, but the new reference counting primitives
    consider this a use after free.  A reference is really not necessary
    here, we are about to deallocate np anyway.
    
    Amends 5eef605e.

diff --git a/nfs/cache.c b/nfs/cache.c
index 2015603..ecf3b11 100644
--- a/nfs/cache.c
+++ b/nfs/cache.c
@@ -129,13 +129,10 @@ netfs_node_norefs (struct node *np)
       args = malloc (sizeof (struct fnd));
       assert (args);
 
-      netfs_nref (np);
-
       args->dir = np->nn->dead_dir;
       args->name = np->nn->dead_name;
       np->nn->dead_dir = 0;
       np->nn->dead_name = 0;
-      netfs_nput (np);
 
       /* Do this in a separate thread so that we don't wait for it; it
 	 acquires a lock on the dir, which we are not allowed to
